desktop-widgets/diveplanner.cpp

```diff
@@ -606,6 +606,8 @@ void PlannerSettingsWidget::setBackgasBreaks(bool dobreaks)
 void PlannerSettingsWidget::setBailoutVisibility(int mode)
 {
 		ui.bailout->setDisabled(!(mode == CCR || mode == PSCR));
+		ui.sacFactor->setDisabled(mode == CCR);
+		ui.sacfactor->setDisabled(mode == CCR);
```

Comment:
Because once is not enough? ;)

Reply:
That way the silly thing is REALLY disabled 🤣

Reply:
I think you could be more trusting with respect to the optimisations modern compilers can do.